syntax = "proto3";

import "google/protobuf/any.proto";
import "attackType.proto";

message GameCommand {
    google.protobuf.Any command = 1;
}

message CommandSetDeck {
    string deck = 1;
}

message CommandReadyToStart {
    bool ready = 1;
}

message CommandDeclareAttack {
    sint32 stageId = 1;
    AttackType attackType = 2;
}

message CommandLevelUp {
    sint32 clockId = 1;
}
